Dockerfiles/agent/entrypoint/89-copy-customfiles.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Copy the custom checks and confs in the /etc/datadog-agent folder
+
+find /conf.d -name '*.yaml' -exec cp --parents {} /etc/datadog-agent/ \;
```

Comment:
Why not using ln -s here ?

Reply:
Is it possible to override an existing check ? In that case the `cp` is obvious. Could we anyway add a `-v` option ?

Reply:
We need to handle subdirectories. For example we might encounter `/conf.d/redisdb.d/myautoconf.yaml`, we'll need to put it in `/etc/datadog-agent/conf.d/redisdb.d/myautoconf.yaml`, but the `/etc/datadog-agent/conf.d/redisdb.d/` folder already exists.

Do you see how we could handle that with `ln` ?

Reply:
Actually I also hope we'll be able to just fetch the configs from `/conf.d` in 6.1 and avoid the copying altogether, as agent6 supports several source paths

Reply:
We could `ln` the directories with a custom depth level ?
Like:
```bash
mkdir -pv /conf.d/redisdb.d/
mkdir: created directory '/conf.d/redisdb.d/'

touch !$myautoconf.yaml
touch /conf.d/redisdb.d/myautoconf.yaml

find /conf.d/ -maxdepth 1 -mindepth 1 -exec ln -svf {} /etc/datadog-agent/ \;
'/etc/datadog-agent/redisdb.d' -> '/conf.d/redisdb.d'

ls -l /etc/datadog-agent/redisdb.d
lrwxrwxrwx 1 root root 17 Feb  8 14:24 /etc/datadog-agent/redisdb.d -> /conf.d/redisdb.d

ls -lH /etc/datadog-agent/redisdb.d
total 0
-rw-r--r-- 1 root root 0 Feb  8 14:20 myautoconf.yaml
```

:100: for
> Actually I also hope we'll be able to just fetch the configs from /conf.d in 6.1